tests/phpunit/tests/test-filters.php

```diff
@@ -363,4 +363,37 @@ public function test_get_terms_inside_query() {
 		add_action( 'pre_get_posts', array( $this, '_action_pre_get_posts' ) );
 		get_posts(); // fires the action and performs the assertions.
 	}
+
+	/**
+	 * @ticket #1682 and Polylang for WooCommerce #939
+	 *
+	 * @see https://github.com/polylang/polylang/issues/1682
+	 * @See https://github.com/polylang/polylang-wc/issues/939
```

Comment:
Let's use the standard format:
```suggestion
	 * @ticket #1682 {@see https://github.com/polylang/polylang/issues/1682}
	 * @ticket #939 {@See https://github.com/polylang/polylang-wc/issues/939}
```